<?xml version="1.0"?>

<launch>
   <arg name="laser_enabled" default="$(optenv HUSKY_LASER_ENABLED true)"/>
   <arg name="world_name" default="$(find husky_highlevel_controller)/worlds/singlePillar.world"/>
   
   <include file="$(find husky_gazebo)/launch/husky_empty_world.launch">
        <arg name="world_name" value="$(arg world_name)"/>
        <arg name="laser_enabled" value="$(arg laser_enabled)"/>
   </include>
  
 
  <node pkg="husky_highlevel_controller" type="husky_highlevel_controller" name="husky_highlevel_controller" output="screen" launch-prefix="gnome-terminal --command">
    <rosparam command="load" file="$(find husky_highlevel_controller)/config/config.yaml" />
  </node>

  <node pkg="rviz" type="rviz" name="rviz" />
</launch>
